react/utils/testUtils.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from 'react'
+import { render } from '@vtex/test-tools/react'
+import { IntlProvider } from 'react-intl'
+
+import * as defaultStrings from '../../messages/en.json'
+
+export const renderWithIntl = (component: ReactElement, options?: any) => {
```

Comment:
Shouldn't this be on the test-tools?

Reply:
This is not necessary! I forgot to remove it 😅